portfolio/src/main/webapp/script.js

```diff
@@ -76,8 +76,12 @@ function fetchComments(limit = '') {
     });
 }
 
+function getRequestParameter(limit, username) {
+  return 'limit=' + limit + (username != null ? '&username=' + username : '');
```

Comment:
Please see comment I made about `undefined` and `null` in #18.

Reply:
It works for checking undefined. Is it better to use typeof?